### PR TITLE
e2e: add days and hours for the e2e tests measure

### PR DESCRIPTION
**Pull request by dougsland**
Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

### CODE REVIEW

File: tests/e2e/run-test-e2e [hunk 1]
```diff
@@ -269,12 +269,14 @@ END_TIME=$(date +%s)
 DURATION=$((END_TIME - START_TIME))
 
 # Calculate minutes and seconds
+DAYS=$((DURATION / 86400))
+HOURS=$(( (DURATION % 86400) / 3600 ))
```
Comment (sandrobonazzola) [inline tests/e2e/run-test-e2e:273]: shouldn't you also update minutes and seconds then?

Comment (dougsland) [reply tests/e2e/run-test-e2e:273]: thanks @sandrobonazzola good catch!

**Review by Yarboa — APPROVED** [review]
LGTM

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Pull request merged**